agiledev-students-spring2024/4-final-project-esports-tracker
# Add additional backend files and implement /feed route with test

Changes Made:
- Added necessary dependencies.
- Provided a simple example of a route and its tests.
- Changed the server to run on port 3001 to avoid conflicts with the front-end running on port 3000.

View Sample Output:
- Pull the latest code.
- Navigate to the back-end directory.
- Run `npm install`.
- Start the server with `npm start`.
- Visit http://localhost:3001/feed (or any other routes you add) to see the sample output.

How to Test:
- Use `npm test` command.
- Alternatively, run `npx c8 mocha tests/feedRoute.test.js` (replace feedRoute with the name of the file you want to test) to test individual files or `npx mocha tests/feedRoute.test.js` if you don't care about the coverage in the moment.

Additional Notes:
- Tests can vary depending on the route's behavior (what it returns, how it handles errors, etc).
- In my example, the back-end and front-end aren't connected yet. Eventually, they should run simultaneously and we should verify that the information is correct.
